tests/python/gaia-ui-tests/gaiatest/apps/system/regions/fxaccounts.py

```diff
@@ -27,11 +40,9 @@ def _switch_to_fxa_iframe(self):
 
     def enter_email(self, email=None):
         self._switch_to_fxa_iframe()
-
-        input_field = Wait(self.marionette, timeout=60).until(
+        email_field = Wait(self.marionette, timeout=60).until(
```

Comment:
Since you're doing this wait also in the init call, you should just use find_element.